rfc/2020/alertmanager-integration/christyjacob4.md

```diff
@@ -0,0 +1,503 @@
+| `June 22nd -  June 29th` | Tests to ensure that `linkerd check` command is functional and update the golden files <br> Configure Alertmanager for High Availability Mode |
+| `June 29th -  July 6th` | Extensive Testing of all the aforementioned functionality <br> Integration Test cases for Alertmanager in  High Availability mode <br> Blog Post #5 |
+| `July 6th -  July 13th` |  Generate Service Profiles for apps with swagger spec, record their per-route metrics and define default Prometheus rules for them  |
+| | *Phase #2 Evaluation Starts `( July 13th )`* |
+| `July 13th -  July 20th` | Generate Service Profiles for apps with protobuf spec, record their per-route metrics and define default Prometheus rules for them <br> Blog Post #6 |
+| `July 20th -  July 27th` | Auto generate Service Profiles for apps without swagger spec or protobuf spec and record their per-route metrics and define default Prometheus rules for them |
+| `July 27th -  August 3rd` | Other Service Profile integration related action items  <br> Blog Post #7 |
+| `August 3rd -  August 10th` | Integration Tests for Service Profile Integration <br> Buffer Time for any action items that may not have been completed |
+| | *Final Evaluation Starts `( August 10th )`* |
+| `August 10th -  August 17th` |   Discuss changes for allowing write access from Linkerd Dashboard <br> Design UI components to support Add/Modify/Delete Prometheus Rules and Alertmanager receivers <br> Blog Post #8 |
+
+
+
+# Unresolved questions
+
+[unresolved-questions]: #unresolved-questions
+
+- I understand that `linkerd profile` can be used to generate Service Profiles based on protobufs, swagger specs, or by using tap to observe the web service. I would like to understand the kind of integration that is expected as part of [Out of the box alerts #1726](https://github.com/linkerd/linkerd2/issues/1726) . Does it mean that the `linkerd profile` command must be automatically run if the user installs Alertmanager, and the generated Service Profile needs to be part of the Alertmanager rules? 
+
+- Should Alertmanager be installed as an Addon module like Trace [Tracing Add-on For Linkerd #3955](https://github.com/linkerd/linkerd2/pull/3955) or should it be a stand alone component like Prometheus, Grafana etc. ?
+
+# Future possibilities
+
+[future-possibilities]: #future-possibilities
+
+- If time permits, work on the identity and RBAC for *Linkerd* web dashboard which will allow modifying the Prometheus rules and Alert Manager Configs directly from the **Linkerd Dashboard**.
+- Allow users to add, delete and modify Prometheus rules using the **Linkerd Dashboard** and inject the new rules into the Prometheus deployment, similar to how `linkerd inject` command works.
+- Allow users to add, delete and modify Alertmanager receivers and also configure them using **Linkerd Dashboard** and inject them.
+- Handling the `linkerd upgrade` scenario and ability to upgrade Alertmanager as part of `linkerd upgrade`.
```

Comment:
`linkerd upgrade` should always upgrade all control plane components and add-ons. Alertmanager shouldn't require some attention.

Reply:
Yep, and this should not be a future possibility as upgrades are essential, and should be completed in the timeline. (As Ivan mentioned shouldn't require anything different for add-on)